### PR TITLE
DAOS-7336 test: Add EC oclass to OSA testing

### PR DESCRIPTION
**Pull request by rpadma2**
Test-tag-hw-large: pr,hw,large offline_extend_oclass
Test-tag-hw-medium: pr,hw,medium,ib2 offline_drain_oclass offline_reintegration_oclass

Summary

- Added EC object class to YAML file for more coverage.

Signed-off-by: rpadma2 <ravindran.padmanabhan@intel.com>

### CODE REVIEW

**Comment by rpadma2** [comment]
daos_offline_drain failure related to DAOS-8609.

**Comment by rpadma2** [comment]
Konwn issue: daos_offline_drain failure related to DAOS-8609.

**Review by shimizukko — APPROVED** [review]

File: src/tests/ftest/osa/offline_drain.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ hosts:
     - server-C
   test_clients:
     - client-D
-timeout: 1100
+timeout: 1200
```
Comment (ravalsam) [inline src/tests/ftest/osa/offline_drain.yaml:10]: The test is timing out in Ci may need more time?

Comment (rpadma2) [reply src/tests/ftest/osa/offline_drain.yaml:10]: In general, the overall test time for different object class testing has increased. So, just adding some more time for handling EC class testing. Present, timeout issue with respect to offline drain is related to DAOS-8609. Di is looking at the issue and feels there is a product issue.

Comment (ravalsam) [reply src/tests/ftest/osa/offline_drain.yaml:10]: So does the test need to be skipped or wait until the defect is resolved?

Comment (rpadma2) [reply src/tests/ftest/osa/offline_drain.yaml:10]: Let me disable the test for now.

**Review by ravalsam — APPROVED** [review]

**Review by shimizukko — APPROVED** [review]

**Comment by sylviachanoiyee** [comment]
@rpadma2 , could you provide instruction in DAOS-8609 at the top of ticket description on how to re-enable test?

**Pull request merged**